docs/features/secret-store/provider/key-per-file.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Since this is not currentlyl released yet, this feature docs should be moved to `/docs/preview/features/secret-store/provider/key-per-file.md`.

Reply:
Togheter with moving it to the `/docs/preview/` path, you may want to add a link to the `/docs/preview/index.md` that pionts to this file.

Reply:
Done.  I've also updated the contents of the documentation file.